package.json

```diff
@@ -84,8 +84,8 @@
     "mocha-webpack": "0.7.0",
     "node-sass": "4.5.2",
     "pre-commit": "1.2.2",
-    "preact": "7.2.1",
-    "preact-compat": "3.14.3",
+    "preact": "8.2.5",
+    "preact-compat": "3.17.0",
```

Comment:
PS: The reason the demo didn't worked with preact was the ws-tile-chart component.
@inanda imported react from the package react which is wrong. 
We have to use the imports.js which handles the switch between preact and react.
I fixed this here https://github.com/fabric-design/components/pull/123/commits/4304d172f949046e84d5164d0f8dee10a3cce29f